src/main/java/eventprocessing/Main.java

```diff
@@ -0,0 +1,79 @@
+import eventprocessing.amazonservices.*;
+import eventprocessing.fileservices.JSONParser;
+import eventprocessing.models.Response;
+import eventprocessing.models.ResponseList;
+import eventprocessing.models.SensorList;
+import eventprocessing.responseservices.ResponseProcessor;
+import eventprocessing.responseservices.ResponseService;
+import eventprocessing.storage.MessageLog;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.IOException;
+import java.util.Scanner;
+
+public class Main {
+    static Logger logger = LogManager.getLogger(Main.class);
+
+    private static ResponseService responseService;
+    private static ResponseProcessor responseProcessor;
+    private static MessageLog messageLog;
+    private static Scanner scanner;
+    private static AmazonController amazonController;
+    private static ResponseList responseList;
+
+    public static void main(String[] args) throws IOException, InterruptedException {
+        logger.debug("App launched");
+        createObjects();
+        JSONParser jsonParser = new JSONParser("locations.json");
+        SensorList sensorList = jsonParser.createSensorList();
+        sensorList.getSensors().forEach(sensor -> System.out.println(sensor.getId()));
+
+        SqsClient sqsClient = amazonController.getSqsClient();
+        String queueUrl = amazonController.getQueueUrl();
+
+        ReceiveMessageRequest receiveMessageRequest = new ReceiveMessageRequest(queueUrl);
+        int counter = 0;
+
+        System.out.println("For how long do you want to run the event processor(seconds)?");
+        int duration = scanner.nextInt();
+
+        while (true) {
+            ReceiveMessageResult messageResult = sqsClient.getSqs().receiveMessage(receiveMessageRequest);
+            for(Message msg : messageResult.getMessages()) {
+                Response response = responseService.parseResponse(msg.getBody());
+                if (responseProcessor.isValidMessage(response, sensorList, messageLog)) {
+                    System.out.println("working sensor with id: " + response.getMessage().getLocationId());
+                    System.out.println(msg.getBody());
+                    responseList.getResponses().add(response);
+                    messageLog.getMessageHistory().add(response.messageId);
+                    messageLog.truncateIfExceedsMaxSize();
+                } else {
+                    System.out.println("sensor not working with id: " + response.getMessage().getLocationId());
+                }
+            }
+            System.out.println("=================================================");
+            Thread.sleep(1000);
+            counter++;
+            if (counter > duration) {
+                sqsClient.destroyQueue();
+                break;
+            }
+        }
+    }
+
+    private static void createObjects() throws IOException {
+        int MAX_SIZE = 300;

```

Comment:
In Java, you tend to only use `SCREAMING_SNAKE_CASE` for constants defined in the body of the class, rather than within a method.

That said, I'd suggest moving this to the config, but it might good to think about whether capping the list by size is the best thing to do - you don't always know what the message volume will be, so you might delete items too soon.